### PR TITLE
[16.0][FIX] stock_account_product_run_fifo_hook: update monkey patches

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Up to date.

### CODE REVIEW

File: stock_account_product_run_fifo_hook/hooks.py [hunk 1]
```diff
@@ -180,6 +177,8 @@ def _run_fifo_vacuum_new(self, company=None):
                     or r.create_date == svl_to_vacuum.create_date
                     and r.id > svl_to_vacuum.id
                 )
+                if self.env.context.get("use_past_svl", False):
+                    candidates = all_candidates
```
Comment (LoisRForgeFlow) [inline stock_account_product_run_fifo_hook/hooks.py:181]: I don't see this upstream https://github.com/odoo/odoo/blob/16.0/addons/stock_account/models/product.py#L433

Comment (MiquelRForgeFlow) [reply stock_account_product_run_fifo_hook/hooks.py:181]: It comes from https://github.com/OCA/stock-logistics-workflow/commit/c9136445025eb15807f796cb38d93594561f5ec6

Comment (LoisRForgeFlow) [reply stock_account_product_run_fifo_hook/hooks.py:181]: OK, I would have put some comment on the manual changes to make it easy to mantain. Anyway, thanks for the update, looks good

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Tested together with stock_valuation_layer_usage module. No issues 👍

**Comment by AaronHForgeFlow** [comment]
Ready to merge IMHO

**Pull request merged**